### ISSUE TITLE
PCKRetrievalTool failed

### ISSUE THREAD

**Issue by zinginging** [intel/SGXDataCenterAttestationPrimitives#301]
Have you tried execute: make clean before you execute command: make STANDALONE=1?

_Originally posted by @jsun39 in https://github.com/intel/SGXDataCenterAttestationPrimitives/issues/225#issuecomment-1101034218_

still got error when `make STANDALONE=1`

Intel(R) Software Guard Extensions PCK Cert ID Retrieval Tool Version 1.16.100.2

Warning: If this is a multi-package platform, please install registration agent package.
         otherwise, the platform manifest information will NOT be retrieved.
Error, call sgx_create_enclave: fail [load_enclave], SGXError:200f.

**Comment by CodingChrisIO** [comment]
I got the same error. 
Running on a Ubuntu 22.04 system with FLC support.

**Comment by LeoneChen** [comment]
Some bugs in PCKRetrievalTool, it uses hard-coded file name
<img width="351" alt="image" src="https://github.com/intel/SGXDataCenterAttestationPrimitives/assets/35480440/73d98402-9a04-41cb-9423-1af7ae18b468">

So I use follow steps:
```
cd /opt/intel/sgx-pck-id-retrieval-tool/
sudo ln -s libsgx_id_enclave.signed.so.1.16.100.1 libsgx_id_enclave.signed.so.1
sudo ln -s libsgx_pce.signed.so.1.19.100.1 libsgx_pce.signed.so.1
```

**Comment by llly** [comment]
Yes. It's a bug. Will fix the `sgx-pck-id-retrieval-tool` package in next release.

**Comment by jsun39** [comment]
@LeoneChen @zinginging what is your Linux distribution?

**Comment by LeoneChen** [comment]
Hi! My kernel version is `5.13.0-39-generic`

**Comment by zinginging** [comment]
> 
Ubuntu 20.04.6 LTS with  kernel 5.14.0

BTW, The os was using LEGACY before, Reinstall the os using UEFI and using command `apt install sgx-pck-id-retrieval-tool`,  `PCKIDRetrievalTool` can execute successfully.

**Comment by bgotowal** [comment]
Closing due to inactivity. An answer and fix have been provided. If you have further questions, please reopen this issue or file a new one.
